app/cdap/components/GridTable/components/Breadcrumb/IngestViewSchemaDropDown/index.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useState } from 'react';
+import { Button, Menu, MenuItem } from '@material-ui/core';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import styled from 'styled-components';
+
+const options = ['Ingest Data', 'View Schema'];
+
+const KebabMenu = styled(Menu)`
+  & div {
+    width: 246px;
+  }
+`;
+
+const KebabIcon = styled(MoreVertIcon)`
+  font-size: 30px;
+`;
+
+export default function() {
```

Comment:
Does this file belong in under `Breadcrumb`?

Reply:
Not exactly. It is a part of the SubHeader component & we have updated the code.